client/src/components/Details/Details.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import { Details } from "./Details.js";
+
+export default {
+  title: "Dropdown/Details",
+  component: Details,
+};
+
+const Template = (args) => <Details {...args} />;
+
+const summary_content = <div>Summary</div>;
+const content = <div>Content</div>;
+
+export const Basic = Template.bind({});
+Basic.args = {
+  is_open: false,
+  summary_content,
+  persist_content: false,
```

Comment:
I think there should be two stories, one for the standard use (`persist_content: false`, content is static), and one for the `persist_content` use case, which will be a little more complicated. To be clear, `persist_content: false` unrenders the children when the dropdown closes, `persist_content: true` just hides them. I.e. the former resets child state when closed and re-opened, the later preserves it.

Maybe this would be best demonstrated by making the child in both cases an incrementing timer?

Reply:
Done! I have implemented this suggestion.